vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -90,6 +90,7 @@ def load_hosts
       def parse_host(host, indexed_servers, _indexed_hosts_ports, indexed_resources)
         uid = host.uuid
         host_name = identify_host_name(indexed_resources, host.instance_uuid, uid)
+        ip_address = identify_primary_ip_address(host, indexed_servers)
```

Comment:
Looks ok to me, this is only setting the ipaddress column on the host, right?  We're using a hostname below when doing the ssh connect, right @Ladas ?